jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/javacore/parser/j9/section/title/TitleTagParser.java

```diff
@@ -45,22 +46,31 @@ protected void initTagAttributeRules() {
 	}
 
 	/**
-	 * Add rule for the dump date (1TIDATETIME line)
+	 * Add rule for the dump date (1TIDATETIME line).
+	 * Also add rule for the dump date (1TIDATETIMEUTC line).
 	 */
 	private void addDateTime() {
 		ILineRule lineRule = new LineRule() {
+			@Override
```

Comment:
There are other missing annotations; see lines 79 and 94.

Reply:
Done